<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>27</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="numLabel">
        <property name="text">
         <string>数    量</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxNum">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
        <property name="suffix">
         <string> Kg</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="priceLabel">
        <property name="text">
         <string>单    价</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="dSpinBoxPrice">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <property name="prefix">
         <string>$ </string>
        </property>
        <property name="maximum">
         <double>9999999999.989999771118164</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="totalLabel">
        <property name="text">
         <string>总    价</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="dSpinBoxTotal">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="prefix">
         <string>$ </string>
        </property>
        <property name="maximum">
         <double>99999999999.990005493164063</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnCalc">
        <property name="text">
         <string>计      算</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>88</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="decLabel">
        <property name="text">
         <string>十  进  制</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxDec">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>22</height>
         </size>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string>Dec </string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnDec">
        <property name="text">
         <string>进 制 转 换</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="binLabel">
        <property name="text">
         <string>二  进  制</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxBin">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>22</height>
         </size>
        </property>
        <property name="prefix">
         <string>Bin </string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="displayIntegerBase">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnBin">
        <property name="text">
         <string>进 制 转 换</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="hexLabel">
        <property name="text">
         <string>十六 进 制</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxHex">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>22</height>
         </size>
        </property>
        <property name="prefix">
         <string>Hex </string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnHex">
        <property name="text">
         <string>进 制 转 换</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
